### PR TITLE
Draft first examples from chapter 03

### PR DESCRIPTION
**Pull request by aloctavodia**
I want to add examples from [Bayesian Statistical Methods](https://www.crcpress.com/Bayesian-Statistical-Methods/Reich-Ghosh/p/book/9780815378648)

### CODE REVIEW

**Comment by canyon289** [comment]
I can take a look at this in next ~48 . hours

**Comment by canyon289** [comment]
Realized this would be hard to review without the book. Ended up buying a copy

**Comment by aloctavodia** [comment]
I am reading the code from [here](https://bayessm.wordpress.ncsu.edu/) (I got the link from the authors).

**Comment by canyon289** [comment]
Quickly looked this over and it LGTM. I like all the color in the python version!

File: BSM/Chapter_03_00_Bayesian_CLT.ipynb [hunk 1]
```diff
@@ -0,0 +1,92 @@
+{
```
Comment (canyon289) [inline BSM/Chapter_03_00_Bayesian_CLT.ipynb]: <p>I like the combining into one. Makes things easier to read</p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

**Pull request merged**